ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,199 @@
+        let fullName = firstName + " " + lastName
+        avatarView.setup(primaryText: fullName, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupAvatarView(with identifier: String) {
+        avatarView.setup(primaryText: identifier, secondaryText: nil, image: avatarImage)
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        avatarView.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(avatarView)
+
+        if let subtitleLabel = subtitleLabel {
+            constraints.append(contentsOf: titleLabelLayoutConstraints())
+            constraints.append(contentsOf: subtitleLabelLayoutConstraints())
+            subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
+            labelContainer.addSubview(subtitleLabel)
+        } else {
+            constraints.append(contentsOf: identifierLayoutConstraints())
+        }
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        labelContainer.addSubview(titleLabel)
+        
+        labelContainer.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(labelContainer)
+
+        constraints.append(contentsOf: labelContainerLayoutConstraints())
+        NSLayoutConstraint.activate(constraints)
+    }
+
+    private func avatarLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            avatarView.heightAnchor.constraint(equalToConstant: Constants.avatarViewHeight),
+            avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewWidth),
+            avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            avatarView.topAnchor.constraint(equalTo: topAnchor),
+            avatarView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            avatarView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ]
+    }
+
+    private func labelContainerLayoutConstraints() -> [NSLayoutConstraint] {
+        return [
+            labelContainer.widthAnchor.constraint(equalToConstant: Constants.contactWidth),
+            labelContainer.centerXAnchor.constraint(equalTo: centerXAnchor),
+            labelContainer.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
+            labelContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            labelContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            labelContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+            labelContainer.heightAnchor.constraint(greaterThanOrEqualToConstant: 2 * Constants.labelMinimumHeight),
+            labelContainer.heightAnchor.constraint(lessThanOrEqualToConstant: Constants.titleLabelMaximumHeight + Constants.subtitleMaximumHeight)
```

Comment:
Why do we need a max height? 
Don't we risk vertically truncating text when the accessibility font size is set to a high value?

Reply:
I originally set a max height because of what I saw in the designs, but your comment is still valid because there are text sizes even larger than what is shown in the designs.